webdriver_wharf/interactions.py

```diff
@@ -193,37 +193,27 @@ def destroy_all():
         destroy(c)
 
 
-def pull(image):
+def pull(image_name):
     global last_pulled_image_id
 
     # Add in some newlines so we can iterate over the concatenated json
-    output = client.pull(image).replace('}{', '}\r\n{')
-    # Check the docker output when running a command, explode if needed
-    for line in output.splitlines():
-        try:
-            out = json.loads(line)
-            if 'error' in out:
-                errmsg = out.get('errorDetail', {'message': out['error']})['message']
-                logger.error(errmsg)
-                # TODO: Explode here if we can't pull...
-                break
-            elif 'id' in out and 'status' in out:
-                logger.debug('{id}: {status}'.format(**out))
-        except:
-            pass
-
-    pulled_image_id = image_id(image)[:12]
+    image = client.images.pull(image_name)
```

Comment:
```python
[INFO] webdriver_wharf.app version 0.3.3.dev3
[INFO] webdriver_wharf.app Pulling image cfmeqe/sel_ff_chrome -- this could take a while
Traceback (most recent call last):
  File "/home/semyers/ve/webdriver-wharf/bin/webdriver-wharf", line 10, in <module>
    sys.exit(main())
  File "/home/semyers/ve/webdriver-wharf/webdriver-wharf/webdriver_wharf/cli.py", line 25, in main
    app.application.try_trigger_before_first_request_functions()
  File "/home/semyers/ve/webdriver-wharf/lib/python2.7/site-packages/flask/app.py", line 1654, in try_trigger_before_first_request_functions
    func()
  File "/home/semyers/ve/webdriver-wharf/webdriver-wharf/webdriver_wharf/app.py", line 360, in _wharf_init
    interactions.pull(image_name)
  File "/home/semyers/ve/webdriver-wharf/webdriver-wharf/webdriver_wharf/interactions.py", line 200, in pull
    image = client.images.pull(image_name)
AttributeError: 'function' object has no attribute 'pull'
```